pkg/kubelet/kubelet.go

```diff
@@ -1408,7 +1408,7 @@ func (kl *Kubelet) setupDataDirs() error {
 	if err := kl.hostutil.MakeRShared(kl.getRootDir()); err != nil {
 		return fmt.Errorf("error configuring root directory: %v", err)
 	}
-	if err := os.MkdirAll(kl.getPodsDir(), 0750); err != nil {
+	if err := utilfs.MkdirAll(kl.getPodsDir(), 0750); err != nil {
```

Comment:
Was this intended to be part of these changes?

Reply:
No I was testing something else and this was left over, removed.